### ISSUE TITLE
Training Loss not Changing  

### ISSUE THREAD

**Issue by uraibeef** [argman/EAST#357]
I training EAST model with my dataset on ICDAR 2015 Format like x1,y1,..,### but in my dataset i set text description to ### for all images (my target is detect all text in receipt) 
OK.. now i'm continue training on my dataset and it can running but training loss not changing look like that
..
Step 000000, model loss 0.0100, total loss 0.0117, 1.50 seconds/step, 2.66 examples/second
Step 000010, model loStep 000000, model loss 0.0100, total loss 0.0117, 1.50 seconds/step, 2.66 examples/second
I try to find bug but i can't found, Can you suggest me some advice? what's wrong?
thank you very much

**Comment by TBourton** [comment]
I think that the samples with '###' are masked out during training: https://github.com/argman/EAST/blob/master/icdar.py#L488

You could try simply changing all of the test descriptions in your training dataset from '###' to 'abc' or some other non-empty text, that way all of the text lines should be included in training.
